docs/stop-server.md

```diff
@@ -4,11 +4,11 @@ Stop a Liberty server. The server instance must exist and must be running.
 
 ###### Additional Parameters
 
-The following are the parameters supported by this goal in addition to the [common server parameters](common-server-parameters.md#common-server-parameters) and the [common parameters](common-parameters.md#common-parameters).
+This goal supports [common server parameters](common-server-parameters.md#common-server-parameters) and [common parameters](common-parameters.md#common-parameters).
 
 | Parameter | Description | Required |
 | --------  | ----------- | -------  |
-| serverStopTimeout | Maximum time to wait (in seconds) to verify that the server has stopped. The default value is 30 seconds. | No |
+| serverStopTimeout | Deprecated. This parameter is ignored. Maximum time to wait (in seconds) to verify that the server has stopped. The default value is 30 seconds. | Ignored |
```

Comment:
So the request is to take out the description.  And change that last "Ignored" back to "No"